code/go/0chain.net/smartcontract/dbs/event/transaction.go

```diff
@@ -134,3 +144,54 @@ func (edb *EventDb) GetTransactionsForBlocks(blockStart, blockEnd int64) ([]Tran
 		Find(&tr)
 	return tr, res.Error
 }
+
+func (edb *EventDb) UpdateTransactionErrors() error {
+
+	var transactions []Transaction
+
+	// created_at for last day from now
+	lastDay := time.Now().AddDate(0, 0, -1)
+	// convert to string
+	lastDayString := lastDay.Format("2006-01-02 15:04:05")
+
+	logging.Logger.Info("lastDayString", zap.Any("lastDayString", lastDayString), zap.Any("lastDay", lastDay), zap.Any("time.Now()", time.Now()))
+
+	// read all the transactions from the transaction table where status is 2 till last day
+
+	err := edb.Get().Model(&Transaction{}).Select("output_hash").Where("status = ? and created_at > ?", 2, lastDayString).Group("output_hash").Find(&transactions)
+
+	if err.Error != nil {
+		logging.Logger.Error("Error while reading transactions from transaction table", zap.Any("error", err.Error))
+		return err.Error
+	}
+
+	logging.Logger.Info("jayashtransactionsLen", zap.Any("transactions", len(transactions)))
```

Comment:
Clean up the logs before merging the PR, could be `logging.Logger.Info("update transaction errors, txn len"...)`.